observability/otel/otel-collector/otel-collector-extensions.adoc

```diff
@@ -158,6 +158,8 @@ include::snippets/technology-preview.adoc[]
 <4> Specifies the directory in which the compactor stores the telemetry data.
 <5> Defines the maximum size of the compaction transaction. To ignore the transaction size, set to zero. If omitted, the default is `+65536+` bytes.
 <6> When set, forces the database to perform an `fsync` call after each write operation. This helps to ensure database integrity if there is an interruption to the database process, but at the cost of performance.
+<7> Buffers the OTLP Exporter data on the local file system.
```

Comment:
This suggestion is outside the primary changes, but consider defining what OTLP equals to the document: OpenTelemetry Protoco

Reply:
Thank you, I've deabbreviated OTLP in the section's text.